modules/core-updates/new-password-policy.md

```diff
@@ -0,0 +1,67 @@
+---
+title: New password policy based on zxcvbn
+menuTitle: New password policy
+---
+
+# New password policy introduced in 8.0.x
```

Comment:
Don't you think it would be more interesting to document this from a system reference standpoint rather from a "how to update your software"? I think it could be overly specific otherwise.

Or perhaps the doc can be split in two: one which explains how the password policy is integrated with customers and employees (for reference), and another one that describes how to update existing code (useful for upgrade only).

Reply:
Does this look better with the new commit ?